Equipment/EquipmentDb/Misc/rings.xml

```diff
@@ -246,6 +246,29 @@
 		</source>
 	</item>
 
+	<item id="12545" phase="1">
+		<info name="Eye of Orgrimmar" type="RING" slot="RING" unique="no" req_lvl="60" item_lvl="60" quality="RARE" boe="no" icon="Inv_jewelry_ring_05.png"/>
```

Comment:
Please set `unique="yes"`, otherwise the sim lets you equip two of them. It is unique in the sense it is a one-time reward, even if it isn't marked as unique in the tooltip ingame.

Please add `faction="HORDE"` since it is otherwise also usable for alliance characters.

Please add `<mutex item_id="12544"/>`, as otherwise Thrall's Resolve can be equipped alongside this ring, which is impossible ingame since you have to choose either.

This ring should be very similar to Thrall's Resolve (except for stats, item id, and mutex item id) so you can check that one out.

Reply:
Resolved :)